tutorial/20_recipes/008_specify_params.py

```diff
@@ -0,0 +1,121 @@
+"""
+.. _specify_params:
+
+Specify Hyperparameters
```

Comment:
It may be easier to understand the recipe title by emphasizing that the hyperparameters are set by the user.

```suggestion
Specify Hyperparameters Manually
```